cf-agent/verify_files.c

```diff
@@ -760,10 +754,33 @@ static bool SaveBufferCallback(const char *dest_filename, void *param, NewLineMo
     return true;
 }
 
-static PromiseResult RenderTemplateMustache(EvalContext *ctx, const Promise *pp, const Attributes *attr,
-                                            EditContext *edcontext, const char *template)
+inline static bool CreateFalseWasSpecified(const Promise *pp)
+{
+    assert(pp != NULL);
+    for (size_t i = 0; i < SeqLength(pp->conlist); i++)
+    {
+        Constraint *cp = SeqAt(pp->conlist, i);
+        if (StringEqual(cp->lval, "create"))
+        {
+            if (StringEqual(cp->rval.item, "false") ||
+                StringEqual(cp->rval.item, "no"))
```

Comment:
You can merge these conditions into one `if`.

Reply:
:+1: